README.md

```diff
@@ -1,5 +1,5 @@
 # Charmed PostgreSQL ROCK
-[![Publish ROCK](https://github.com/canonical/charmed-postgresql-rock/actions/workflows/publish.yaml/badge.svg)](https://github.com/canonical/charmed-postgresql-rock/actions/workflows/publish.yaml)
+[![Release to GitHub Container Registry](https://github.com/canonical/charmed-postgresql-rock/actions/workflows/release.yaml/badge.svg)](https://github.com/canonical/charmed-postgresql-rock/actions/workflows/release.yaml)
```

Comment:
IMHO, you can shortcut `Release to GitHub Container Registry` to `Release to GHCR`, `GHCR Release` or `Release to GHCR.io`. It will be too long badge on https://releases.juju.is/?team=Data